src/main.tsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
```

Comment:
When I reload any page except the Home, I get 404 error. Because GitHub Pages just gives static pages, and there is no such page as `/products.html`
The solution here is to use [Hash router](https://reactrouter.com/en/main/router-components/hash-router) instead of Browser one 
<img width="1184" alt="image" src="https://github.com/roman-orlovskyi-moc/fe-react-2024/assets/98031768/3f69e048-0ebc-463a-bc9d-969c1b107a73">

Reply:
fixed